test/unit_tests/apps/regulations3k/js/module2-spec.js

```diff
@@ -1,13 +1,11 @@
 const BASE_JS_PATH = '../../../../../cfgov/unprocessed/apps/regulations3k';
 
-const expect = require( 'chai' ).expect;
-
 const module = require( `${ BASE_JS_PATH }/js/module2.js` );
 
-describe( 'Some other module that does somthing', function() {
+describe( 'Some other module that does somthing', () => {
```

Comment:
While you're in there, please fix my typo. 🙊 `somthing` -> `something`

Reply:
Fixed!